### PR TITLE
Fix: Fixed issue where GridSplitter style was missing on the details layout

### LINKED ISSUE #9927: Bug: Details grid is missing the grid splitter style when resizing columns
**Requirements**
- Add back grid splitter style when resizing DataGrid columns

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9927 

**Comments**
It seems this is a <a href="https://github.com/microsoft/microsoft-ui-xaml/issues/4834">known bug</a> of WinUI3.
This is a workaround to fix that.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs [hunk 1]
```diff
@@ -31,6 +33,9 @@ public sealed partial class DetailsLayoutBrowser : BaseLayout
 	{
 		private uint currentIconSize;
 
+		private InputCursor arrowCursor = InputCursor.CreateFromCoreCursor(new CoreCursor(CoreCursorType.Arrow, 0));
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:36]: Should add an empty line between the two fields.

Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:36]: Should be an empty line between the two fields declaration.

**Review by QuaintMako — COMMENTED** [review]
Aside for the empty line to add, code LGTM!

Good idea to reorder the imports in alphabetical order.

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**